spec/features/solr_search_page_spec.rb

```diff
@@ -3,43 +3,66 @@
 RSpec.describe "Solr Search page", type: :feature do
   let(:results) { File.read(Rails.root.join("spec/fixtures/solr_response.json").to_s) }
 
-  before do
-    allow(Search::Solr).to receive(:search).and_return(JSON.parse(results))
-  end
+  describe "Default search page behaviour" do
+    before do
+      allow(Search::Solr).to receive(:search).and_return(JSON.parse(results))
+    end
 
-  scenario "Displays search heading" do
-    visit "/search/solr"
-    expect(page).to have_css("h1", text: "Search results")
-  end
+    scenario "Displays search heading" do
```

Comment:
Is this indent intentional?

Reply:
Yes I've grouped these tests into a separate `describe` block to keep them separate from the new tests added.